BREAKING.md

```diff
@@ -1,3 +1,6 @@
+## Normalizing enum ContainerErrorType
+In an effort to clarify error categorization, a name and value in this enumeration were changed.
```

Comment:
this should go in the 0.33 section, as thats the the current master version. the pattern is to put a link to section at the beginning of the versions section, and then append the change's section